src/javascript/app_2/pages/trading/components/elements/tabs.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+class Tabs extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = { active: '1' };
+    }
+    render() {
+        const className = 'tab-container';
+        return (
+            <div className={`${className} ${this.props.alignment}`}>
+                <TabsWrapper
+                  active={this.state.active}
+                  onChange={active => this.setState({active})}
+                >
+                {Object.keys(this.props.list).map(key => (
+                    <React.Fragment key={key}>
+                        {!!this.props.list[key].icon &&
+                            <i className={this.props.list[key].icon}></i>
+                        }
+                        <span>{this.props.list[key].header}</span>
+                    </React.Fragment>
+                ))}
+                </TabsWrapper>
+                <p className='tab-content'>{this.props.list[this.state.active].content}</p>
+            </div>
```

Comment:
nitpick: `active` sounds like a boolean, maybe `activeIdx`?

Reply:
Alright, will change it. :)

Reply:
`active_tab_index` will be better, no?

Reply:
Yea, sure 👍